content/app/development/configuration/eformidling/_index.en.md

```diff
@@ -0,0 +1,286 @@
+you can overide this configuration in  _appsettings.Development.json_.
+
+Create the section _AppSettings_, if it does not already exist, and set _EnableEFormidling_ to false.
+
+```json
+"AppSettings": {
+    "EnableEFormidling": false
+}
+```
+
+## Adding support for eForimdling in App.cs
+
+The next step in setting up support of eFormidling , 
+is to make the required services available for the appliction.
+All changes are made in _App.cs_, which you fill find in the _App/logic folder.
+
+At the top of the file, among the library references, include the following three lines.
+
+```cs
+using Altinn.Common.EFormidlingClient.Models;
+using Altinn.Common.EFormidlingClient;
+using Altinn.Common.AccessTokenClient.Services;
+```
+
+Further, you need to inject services in the constructor of both the class and the base class.
+
+Before making any changes, the constructor should look like the example below, 
+but which services are included may vary from application to application.
+This shows the most common setup.
+
+
+```cs
+public App(
+IAppResources appResourcesService,
+(...)
+IHttpContextAccessor httpContextAccessor):base(
+appResourcesService,
+(...)
+httpContextAccessor)
+```
+
+The list of services in the constructor should be extended with the four services listed below.
+
+```cs
+IEFormidlingClient eformidlingClient,
+IOptions<AppSettings> appsettings,
+IAccessTokenGenerator tokenGenerator,
+IOptions<PlatformSettings> platformSettings
+```
+
+Further, these services should be forwarded to the base class as well, 
+but in this case only include the names, not the types.
+
+
+```cs
+eformidlingClient,
+appsettings,
+platformSettings,
+tokenGenerator
+```
+
+Final result should look like this:
+
+```cs
+public App(
+IAppResources appResourcesService,
+(...)
+IHttpContextAccessor httpContextAccessor,
+IEFormidlingClient eformidlingClient,
+IOptions<AppSettings> appsettings,
+IAccessTokenGenerator tokenGenerator,
+IOptions<PlatformSettings> platformSettings):base(
+appResourcesService,
+(...)
+httpContextAccessor,
+eformidlingClient,
+appsettings,
+platformSettings,
+tokenGenerator)
+```
+
+## Configuring key values for eFormidling in your application
+
+Metadata related to the eFormidling shipment is required, 
+and this is set up in _applicationmetadata.json_.
+You find the file in the _App/config_ folder.
+
+Create a new section `eFormidling` and populate values for the 
+parameters defined in the table.
+
+| Id              | Description                                                                                                |
+| --------------- | ---------------------------------------------------------------------------------------------------------- |
+| serviceId       | Id specifying shipment type DPO, DPV, DPI eller DPF*                                                       |
+| process         | Id which wil be included in the scope of the StandardBusinessDocumentHeader**                              |
```

Comment:
Har e-formidling noen egen dokumentasjon på dette med prosess som vi kunne lenket til?

Reply:
Finner ingen side som tar for seg prosessene utelukkende, men har lenket inn doc til meldingstypene der prosess står innunder hvert alternativ